src/components/Accordion/Root.tsx

```diff
@@ -1,29 +1,58 @@
 import { Root as AccordionRoot } from "@radix-ui/react-accordion";
-import { forwardRef, ReactNode } from "react";
+import { forwardRef } from "react";
 import { Box, PropsWithBox } from "../Box";
 
-export type AccordionRootProps = PropsWithBox<{
-  children: ReactNode;
+interface SingleProps {
+  type: "single";
   defaultValue?: string;
   value?: string;
   onValueChange?: (value: string) => void;
-}>;
+}
+
+interface MultipleProps {
+  type: "multiple";
+  defaultValue?: string[];
+  value?: string[];
+  onValueChange?: (value: string[]) => void;
+}
+
+export type AccordionRootProps = PropsWithBox<MultipleProps | SingleProps>;
 
 export const Root = forwardRef<HTMLElement, AccordionRootProps>(
-  ({ children, defaultValue, value, onValueChange, ...rest }, ref) => (
-    <AccordionRoot
-      type="single"
-      collapsible
-      defaultValue={defaultValue}
-      value={value}
-      onValueChange={onValueChange}
-      asChild
-    >
+  ({ children, defaultValue, value, onValueChange, type, ...rest }, ref) => {
```

Comment:
the type parameter is a BC break as becomes required now - why not apply the default value as "single"? what's the benefit having this required?